outputs.tf

```
@@ -29,6 +29,12 @@ output "reader_endpoint" {
   description = "A read-only endpoint for the Aurora cluster, automatically load-balanced across replicas"
 }
 
+# See: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster.html#master_user_secret
+output "admin_user_secret" {
+  value       = var.manage_admin_user_password ? join("", aws_rds_cluster.primary[*].master_user_secret) : []
```

Hi @thedomeffm 

I missed this yesterday but since `master_user_secret` is a list, we'll need to remove the join. I think this should work but haven't tested:

```suggestion
  value       = var.manage_admin_user_password ? concat(aws_rds_cluster.primary[*].master_user_secret) : []
```